### ISSUE TITLE
Add ability for specifying the platform version on update-service with CodeDeploy

### ISSUE THREAD

**Issue by dhaspden** [CircleCI-Public/aws-ecs-orb#107]
## What would you like to be added

It would be nice to have a `platform-version` parameter on the `update-service` command so that we can specify a custom platform version for our CodeDeploy deployments. An example would look like the following:

```yml
- aws-ecs/update-service:
    # ...
    platform-version: "1.4.0"
```

I guess you could default this to `LATEST` or whatever a sensible default is. I'm not familiar with ECS on EC2 so I'm not sure how the platform version behaves there. I am coming from a Fargate background so 1.4.0 is the newest version.

## Why is this needed

Currently it isn't possible to update the platform version using this orb due to the way it generates the AppSpec. This would provide an easier method to update from older platform versions to newer platform versions.

**Comment by alekhrycaiko** [comment]
From what I've read @dhaspden the `platform-version` is a purely fargate feature. It'd probably make the most sense to add this command with that consideration in mind.

**Comment by dhaspden** [comment]
From a CodeDeploy perspective it would be nice to somehow be able to add extra fields to the generated json. If it can be made generic to support platform version as well as whatever other field folks want (like hooks)? I just know since it defaults to `LATEST` which is actually `1.3.0` right now it blocks us from being able to use this orb _and_ deploy `1.4.0`. Not sure if there are any known workarounds though.

**Comment by reytarovskiy** [comment]
+1

**Comment by LeeCheneler** [comment]
So our Fargate deployments broke last week because our services need platform version 1.3.0 and LATEST now points to 1.4.0 in AWS.  It looks like we need to ditch this orb and use our own scripts now because of this.

+1 for adding support to set the platform version... please consider it!

**Comment by brivu** [comment]
Hey everyone, 

This issue has been resolved with PR #118. I am going to close this issue for now. If you still have any outstanding issues, please open a new one and we'll look into it. 

Best, 
Brian
